lib/diagnostic.js

```diff
@@ -32,7 +47,9 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
-
+db.houses.update({name: "House Stark"}, {$push: {["members": "Ned Stark", "Arya Stark", "Sansa Stark"]}});
```

Comment:
`$push`'s value is an invalid object.  Here is a solution using a combination of `$set`, `$push`, and `$push` with `$each`.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```